apollo-router/src/plugins/authentication/mod.rs

```diff
@@ -589,12 +589,6 @@ fn authenticate(
         status: StatusCode,
     ) -> ControlFlow<router::Response, router::Request> {
         // This is a metric and will not appear in the logs
-        u64_counter!(
-            "apollo_authentication_failure_count",
-            "Number of requests with failed JWT authentication (deprecated)",
-            1,
-            kind = "JWT"
-        );
```

Comment:
I removed several of these metrics that had `deprecated` notes. Let me know if this shouldn't happen.